src/packages/client/src/generation/TSClient.ts

```diff
@@ -86,74 +84,75 @@ const {
 const path = require('path')
 const debug = debugLib('prisma-client')
 
+const Prisma = {}
+
+exports.Prisma = Prisma
+
 /**
  * Prisma Client JS version: ${clientVersion}
  * Query Engine version: ${engineVersion}
  */
-exports.prismaVersion = {
+Prisma.prismaVersion = {
   client: "${clientVersion}",
   engine: "${engineVersion}"
 }
 
-exports.PrismaClientKnownRequestError = PrismaClientKnownRequestError;
-exports.PrismaClientUnknownRequestError = PrismaClientUnknownRequestError;
-exports.PrismaClientRustPanicError = PrismaClientRustPanicError;
-exports.PrismaClientInitializationError = PrismaClientInitializationError;
-exports.PrismaClientValidationError = PrismaClientValidationError;
-exports.Decimal = Decimal
+Prisma.PrismaClientKnownRequestError = PrismaClientKnownRequestError;
+Prisma.PrismaClientUnknownRequestError = PrismaClientUnknownRequestError;
+Prisma.PrismaClientRustPanicError = PrismaClientRustPanicError;
+Prisma.PrismaClientInitializationError = PrismaClientInitializationError;
+Prisma.PrismaClientValidationError = PrismaClientValidationError;
+Prisma.Decimal = Decimal
 
 /**
  * Re-export of sql-template-tag
  */
 
-exports.sql = sqltag
-exports.empty = empty
-exports.join = join
-exports.raw = raw
+Prisma.sql = sqltag
+Prisma.empty = empty
+Prisma.join = join
+Prisma.raw = raw
 `
 
 const commonCodeTS = ({
   runtimePath,
   clientVersion,
   engineVersion,
-}: CommonCodeParams): string => `import {
-  DMMF,
-  DMMFClass,
-  Engine,
-  PrismaClientKnownRequestError,
-  PrismaClientUnknownRequestError,
-  PrismaClientRustPanicError,
-  PrismaClientInitializationError,
-  PrismaClientValidationError,
-  sqltag as sql,
-  empty,
-  join,
-  raw,
-  Sql,
-  Decimal,
-} from '${runtimePath}';
+}: CommonCodeParams): { ts: string, tsWithoutNamespace: string } => ({
+  tsWithoutNamespace: `import * as runtime from '${runtimePath}';`, ts: `export import DMMF = runtime.DMMF
```

Comment:
```suggestion
  tsWithoutNamespace: `import * as runtime from '${runtimePath}';`,
  ts: `export import DMMF = runtime.DMMF
```
I kind of missed that detail because not on a new line 😄